src/routes/card/[id]/+page.server.ts

```
@@ -0,0 +1,24 @@
+import type { PageServerLoad } from './$types';
+import { getCardById, getSkillDetails } from '$lib/db';
+import { error } from '@sveltejs/kit';
+
+export const load: PageServerLoad = async ({ params }) => {
+  const id = parseInt(params.id);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)